roles/kube-burner/files/metrics.yaml

```diff
@@ -1,11 +1,11 @@
 metrics:
-  - query: sum(irate(container_cpu_usage_seconds_total{name!="",namespace=~"openshift-(etcd|oauth-apiserver|.*apiserver|ovn-kubernetes|sdn|ingress|authentication|.*controller-manager|.*scheduler)"}[2m]) * 100) by (pod, namespace, node)
+  - query: sum(irate(container_cpu_usage_seconds_total{name!="",namespace=~"openshift-(etcd|oauth-apiserver|.*apiserver|ovn-kubernetes|sdn|ingress|authentication|.*controller-manager|.*scheduler|monitoring|logging|image-registry)"}[2m]) * 100) by (pod, namespace, node)
```

Comment:
This will generate lot of documents on a large scale cluster since there's going to be a replica of node-exporter, fluentd ( when logging is installed ) on each of the nodes.

Reply:
Adding specific queries to grab the resource usage of prometheus and elasticsearch pods ( if deployed ) might be the way to go. Thoughts?

Reply:
Looks good to me, you're right about the high number of documents generated, however, the metrics.yaml is meant to be used in smaller scale scenarios (i.e node-density), so I think your suggestion of adding a prometheus container specific metric is a good choice in metrics-aggregated.yaml.

Reply:
Makes sense.